aspnet/mvc/views/overview.rst

```diff
@@ -0,0 +1,79 @@
+
+How do Controllers Specify Views?
+---------------------------------
+
+Views are typically returned from actions as a :dn:cls:`~Microsoft.AspNetCore.Mvc.ViewResult`. Your action method can create and return a ``ViewResult`` directly, but more commonly if your controller inherits from :dn:cls:`~Microsoft.AspNetCore.Mvc.Controller`, you'll simply use the ``View`` helper method, as this example demonstrates:
+
+.. literalinclude:: /../common/samples/WebApplication1/src/WebApplication1/Controllers/HomeController.cs
+  :language: c#
+  :lines: 9-14
+  :emphasize-lines: 5
+  :dedent: 4
+  
+The ``View`` helper method has several overloads to make returning views easier for app developers. You can optionally specify a specific view to return, as well as a model to pass to the view. With no parameters, the default convention of using a view name that corresponds to the action name is used.
+
+View Discovery
+^^^^^^^^^^^^^^
+
+When an action returns a view, a process called *view discovery* takes place. This process determines which view file will be used. There are several scenarios, based on how the view was specified from the action method.
+
+``return View();``
+    The most common approach. In this case the framework will use a view with the same name as the action (with a *.cshtml* extension) located in a folder named the same as the controller. Otherwise, it will look in the *Shared* folder for a view with a name corresponding to the action name.
+
+``return View("List");``
+    Using this syntax, a view with the provided name (and a *.cshtml* extension) is used instead of the current action name. Otherwise, the behavior is as above.
+
+``return View("Views/Home/About.cshtml");``
+    Specify the path from the application root to the view. Using this syntax, the *.cshtml* extension must be specified. The path can optionally start with "/" or "~/".
+
+.. note:: :doc:`Partial views <partial>` and :doc:`view components <view-components>` use similar (but not identical) discovery mechanisms.
+
+.. note:: You can customize the default convention regarding where views are located within the app by using a custom :dn:iface:`~Microsoft.AspNetCore.Mvc.Razor.IViewLocationExpander`.
+
+.. note:: View names may be case sensitive depending on the underlying file system. Case insensitive file systems can specify "about" or "About" or "about.cshtml" or "About.cshtml" to refer to the file *About.cshtml*. However, case-sensitive systems must use case-sensitive mathcing files/folders corresponding to view, action, and controller names.
+
+Passing Data to Views
+^^^^^^^^^^^^^^^^^^^^^
+
+You can pass data to views using several mechanisms. The most robust approach is to specify a *model* type in the view, and then pass an instance of this type to the view from the action. This allows the view to take advantage of strong type checking, and is the recommended approach for most scenarios.
```

Comment:
We recommend you use a model or view model to pass data to a view. This allows the view to take advantage of strong type checking.